troubleshooting/general-issues.md

```diff
@@ -23,6 +24,115 @@ This page provides solutions for common issues you may encounter while working w
 * [NuGet Feed Troubleshooting]({%slug troubleshooting-nuget%})
 * [Upload Troubleshooting]({%slug upload-troubleshooting%})
 
+## TelerikRootComponent is missing in .NET 8.0
+
+The latest UI for Blazor (4.6.0) provides [compatibility with .NET 8.0 RC 2]({%slug system-requirements%}#compatible-net-versions). This framework version [is not yet supported]({%slug system-requirements%}#supported-net-versions) and we do not recommend using our components in it until officially supported.
+
+If you need to start testing with .NET 8.0 RC 2 at this stage, be aware of the following error you may hit when using the new [Blazor Web App template](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-8-rc-1/#blazor-web-app-template-updates)
+
+>warning Error: System.Exception: A Telerik component on the requested view requires a TelerikRootComponent to be added to the root of the MainLayout component of the app.
+
+The root cause for this is a difference in the required configuration when [interactive render modes](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#enable-support-for-interactive-render-modes) are used.
+
+.NET 8.0 introduces [new render modes for the Blazor components](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0). It is required that the `TelerikRootComponent` is placed in a layout page (e.g. `MainLayout.razor`) that has interactive mode enabled. At the time of writing,the default render mode is `Static` and not interactive, so you need to make this change explicitly in your app.
+
+Here are two options to go ahead with:
+
+* Set the render mode for the whole app as suggested in the [Blazor documentation](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#set-the-render-mode-for-the-entire-app). This will save you specifying the render mode in every page and component.
```

Comment:
```suggestion
* Set the render mode for the whole app as suggested in the [Blazor documentation](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0#set-the-render-mode-for-the-entire-app). This will spare the need to set the render mode in every page and component.
```